drivers/bus/acpi/cmbatt/cmbatt.c

```diff
@@ -292,16 +294,73 @@ CmBattVerifyStaticInfo(PCMBATT_DEVICE_EXTENSION DeviceExtension,
         // FIXME: take from _BIX method: Info->CycleCount
         DeviceExtension->BifData = BifData;
 
-        if (BifData.PowerUnit == 1)
+        /* Check if the power stats are reported in ampere or watts */
+        if (BifData.PowerUnit == ACPI_BATT_POWER_UNIT_AMPS)
         {
```

Comment:
You can move `ULONG DesignVoltage` definition to this code block, or declare it right on the line where it's assigned.

Reply:
No.

Reply:
Are you going to reuse DesignVoltage later outside of that code block?